modules/openapi-generator/src/main/resources/php-symfony/.coveralls.yml

```diff
@@ -0,0 +1,6 @@
+# same as Ruby lib
```

Comment:
Shall it be PHP lib? I don't how Ruby is related in this PR.

Reply:
Yeah, looks weird, but it's just a comment. I've blindly copied it from official [.coveralls.yml example configuration](https://github.com/php-coveralls/php-coveralls#coverallsyml) few years ago. It works, trust me. :smile:

Reply:
I can remove that comment to avoid misunderstanding.

Reply:
Yes please as someone else may also find this confusing.